tests/licensedcode/data/licenses/url_badge.yml

```diff
@@ -0,0 +1,26 @@
+license_expressions:
+  -agpl-3.0
```

Comment:
you likely want a space after each dash as in 
`    - agpl-3.0`

Reply:
- Yes, the scan is detecting the new rules
- Fixed indentation of url_badge.yml